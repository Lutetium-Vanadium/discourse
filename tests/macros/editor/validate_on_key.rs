fn main() {
    requestty::questions![Editor { validate_on_key: todo!() }];
}
