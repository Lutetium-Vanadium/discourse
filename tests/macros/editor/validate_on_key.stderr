error: option `validate_on_key` does not exist for kind `editor`
 --> tests/macros/editor/validate_on_key.rs:2:36
  |
2 |     requestty::questions![Editor { validate_on_key: todo!() }];
  |                                    ^^^^^^^^^^^^^^^
